client/jetpack-connect/auth-form-header.jsx

```diff
@@ -189,20 +190,36 @@ export class AuthFormHeader extends Component {
 				case 'logged-out':
 					return config.isEnabled( 'woocommerce/core-profiler-passwordless-auth' )
 						? translate(
-								"We'll make it quick – promise. In order to take advantage of the benefits offered by %(pluginName)s, you'll need to create a WordPress account. {{br/}} Already have one? {{a}}Log in{{/a}}",
-								translateParams
+								'To access all of the features and functionality in %(pluginName)s, you’ll first need to connect your store to a WordPress.com account. Please create one now, or {{a}}log in{{/a}}. {{br}}{{/br}}For more information, please {{doc}}review our documentation{{/doc}}.',
```

Comment:
Could we move the conditional up to the `case` instead of using a ternary? I think it would be easier to read. Alternatively, putting up a whole `if` branch above the switch and duplicating the switch would be too. This also has the benefit of making it easy to remove the feature flag later on by just deleting the conditional without having to untangle it.

Reply:
@rjchow Unfortunately, we have other places where I used a ternary operator to check the feature flag. I think we would need to update all of them and run the tests again if we want to switch to using if statements instead. At this point, I feel like it’s not worth retesting everything for this change.

I still think it would be easy to remove later if I’m not misunderstanding your suggestion.

```
return config.isEnabled( 'woocommerce/core-profiler-passwordless-auth' )
	? translate(
			'To access all of the features and functionality in %(pluginName)s, you’ll first need to connect your store to a WordPress.com account. Please create one now, or {{a}}log in{{/a}}. {{br}}{{/br}}For more information, please {{doc}}review our documentation{{/doc}}.',
			{
				...translateParams,
				components: {
					...translateParams.components,
					doc: reviewDocLink,
				},
			}
	  )
	: translate(
			"We'll make it quick – promise. In order to take advantage of the benefits offered by %(pluginName)s, you'll need to connect your store to your WordPress.com account. {{br/}} Already have one? {{a}}Log in{{/a}}",
			translateParams
	  );
```

to

```
return translate(
	'To access all of the features and functionality in %(pluginName)s, you’ll first need to connect your store to a WordPress.com account. Please create one now, or {{a}}log in{{/a}}. {{br}}{{/br}}For more information, please {{doc}}review our documentation{{/doc}}.',
	{
		...translateParams,
		components: {
			...translateParams.components,
			doc: reviewDocLink,
		},
	}
)
```

should be fairly straightforward.

Reply:
Here's what I had in mind: https://github.com/Automattic/wp-calypso/compare/update/passwordless-auth-copy-changes...update/passwordless-auth-copy-changes-refactor

It's logically equivalent so an inspection and reasoning through it should be sufficient to confirm that it will work the same, but the benefit is that later on when it comes time to remove the feature flag you can delete the entire contiguous block without having to strain about whether anything was missed out due to the two different branches being interleaved.

In this instance you may feel that it will take longer to resolve this disagreement than to disentangle the code - but frequently these structures persist and more code gets tacked on from this state. 

I'd just like us to be a bit more mindful about the future when we commit code, as code is read a lot more than it is written

Reply:
@rjchow I would agree with you if we were planning to run an experiment with this feature flag and keep it around for weeks or months. However, this feature flag will be removed as soon as we're ready to push the code to production. It was placed temporarily for testing on Horizon. I really think it's just a 5-second job to remove this feature flag, and it's not confusing at all, at least for now.

I do agree that the if...else style is better when we have more lines of code though. I don't have a strong preference, to be honest. I just figured it's not an important piece of code to change, given that we'll remove it after the testing period before pushing it to production.

Updated it in [Use if..else style for the feature flag](https://github.com/Automattic/wp-calypso/pull/94963/commits/1baa97ed04efabc6d269d62a4da6070b79a19060)

Reply:
Thanks @moon0326, I didn't mean to hold this up - appreciate the consideration

Reply:
> Thanks @moon0326, I didn't mean to hold this up - appreciate the consideration

No worries! Thanks for the suggestion 👍 I'll start using that style from now on.